backend/adminRouter.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const Students = require('../models/students');
+
+const adminRouter = express.Router();
+
+dishRouter.use(bodyParser.json());
+
+adminRouter.route('/')
+/*.get((req,res,next) => {
+    Students.find({})
+    .then((students) => {
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'application/json');
+        res.json(students);
+    }, (err) => next(err))
+    .catch((err) => next(err));
+})*/
+.post((req, res, next) => {
+    Students.create(req.body)
+    .then((student) => {
+        console.log('Student record created', student);
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'application/json');
+        res.json(student);
+    }, (err) => next(err))
+    .catch((err) => next(err));
+});
+.delete((req, res, next) => {
+    Students.remove(req.body)
+    
+    }, (err) => next(err))
+    .catch((err) => next(err));
+});
+
```

Comment:
Should we not allow admins to delete all student records at once?
@BojithaPiyathilake @Nifla1998  are you going to request a delete method for all records at once? If not no need of this part I guess.
```suggestion can remove the following part
.delete((req, res, next) => {
    Students.remove(req.body)
    
    }, (err) => next(err))
    .catch((err) => next(err));
});

```